meilisearch-http/src/helpers/meilisearch.rs

```diff
@@ -223,12 +223,12 @@ impl<'a> SearchBuilder<'a> {
         }
 
         let start = Instant::now();
-        let docs =
-            query_builder.query(reader, &self.query, self.offset..(self.offset + self.limit));
+        let result = query_builder.query(reader, &self.query, self.offset..(self.offset + self.limit));
         let time_ms = start.elapsed().as_millis() as usize;
+        let (docs, nb_hits) = result.map_err(|e| Error::SearchDocuments(e.to_string()))?;
```

Comment:
I would prefer to keep those times lines together.

```suggestion
        let result = query_builder.query(reader, &self.query, self.offset..(self.offset + self.limit));
        let (docs, nb_hits) = result.map_err(|e| Error::SearchDocuments(e.to_string()))?;
        let time_ms = start.elapsed().as_millis() as usize;
```